x-pack/plugins/triggers_actions_ui/public/application/sections/alert_form/alert_notify_when.tsx

```diff
@@ -0,0 +1,220 @@
+          <FormattedMessage
+            defaultMessage="Run every time alert is active"
+            id="xpack.triggersActionsUI.sections.alertForm.alertNotifyWhen.onActiveAlert.label"
+          />
+        </strong>
+        <EuiText size="s" color="subdued">
+          <p>
+            <FormattedMessage
+              defaultMessage="Actions run with every active alert interval."
+              id="xpack.triggersActionsUI.sections.alertForm.alertNotifyWhen.onActiveAlert.description"
+            />
+          </p>
+        </EuiText>
+      </Fragment>
+    ),
+  },
+  {
+    value: 'onThrottleInterval',
+    inputDisplay: 'Set a custom action interval',
+    'data-test-subj': 'onThrottleInterval',
+    dropdownDisplay: (
+      <Fragment>
+        <strong>
+          <FormattedMessage
+            defaultMessage="Set a custom action interval"
+            id="xpack.triggersActionsUI.sections.alertForm.alertNotifyWhen.onThrottleInterval.label"
+          />
+        </strong>
+        <EuiText size="s" color="subdued">
+          <p>
+            <FormattedMessage
+              defaultMessage="Actions run using the interval you set."
+              id="xpack.triggersActionsUI.sections.alertForm.alertNotifyWhen.onThrottleInterval.description"
+            />
+          </p>
+        </EuiText>
+      </Fragment>
+    ),
+  },
+];
+
+interface AlertNotifyWhenProps {
+  alert: InitialAlert;
+  throttle: number | null;
+  throttleUnit: string;
+  onNotifyWhenChange: (notifyWhen: AlertNotifyWhenType) => void;
+  onThrottleChange: (throttle: number | null, throttleUnit: string) => void;
+}
+
+export const AlertNotifyWhen = ({
+  alert,
+  throttle,
+  throttleUnit,
+  onNotifyWhenChange,
+  onThrottleChange,
+}: AlertNotifyWhenProps) => {
+  const [alertThrottle, setAlertThrottle] = useState<number>(throttle || 1);
+  const [showCustomThrottleOpts, setShowCustomThrottleOpts] = useState<boolean>(false);
+  const [notifyWhenValue, setNotifyWhenValue] = useState<AlertNotifyWhenType>(
+    DEFAULT_NOTIFY_WHEN_VALUE
+  );
+
+  useEffect(() => {
+    if (alert.notifyWhen) {
+      setNotifyWhenValue(alert.notifyWhen);
+    } else {
+      // If 'notifyWhen' is not set, derive value from existence of throttle value
+      setNotifyWhenValue(alert.throttle ? 'onThrottleInterval' : 'onActiveAlert');
+    }
+  }, [alert]);
+
+  useEffect(() => {
+    setShowCustomThrottleOpts(notifyWhenValue === 'onThrottleInterval');
+  }, [notifyWhenValue]);
+
+  const onNotifyWhenValueChange = useCallback((newValue: AlertNotifyWhenType) => {
+    onThrottleChange(newValue === 'onThrottleInterval' ? alertThrottle : null, throttleUnit);
+    onNotifyWhenChange(newValue);
+    setNotifyWhenValue(newValue);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  const labelForAlertRenotify = (
+    <>
+      <FormattedMessage
+        id="xpack.triggersActionsUI.sections.alertForm.renotifyFieldLabel"
+        defaultMessage="Notify every"
```

Comment:
I saw in @mdefazio's screenshots that the "Notify every" label changed to "Action frequency" (I haven't followed the PR to see if this was discussed already).

To me, it seems to read better with the new label: `Action frequency: Run only..`.

I don't want this question to block the PR so this can be a follow up issue / change if it wasn't previously discussed.